functions/supportfiles/download-eeglab-plugin

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+#
+# Kevin Murphy
+# kevin@murp.us
+
+set -euo pipefail
+
+if [[ $# -ne 1 ]]; then
+    cat <<USAGE >&2
+usage: $0 <plugin-name>
+
+    Install a plugin called <plugin-name> from the sccn.ucsd.edu website.
+
+USAGE
+    exit 1
+fi
+
+# Go to the root of this repository so that matlab imports work correctly.
+cd "$(git rev-parse --show-toplevel)"
+
+PLUGIN_BASE_URL="https://sccn.ucsd.edu/eeglab/plugin_uploader/plugin_list_all.php"
+PLUGIN_NAME="$1"
+
+# Use python to query the sccn.ucsd.edu plugin website for the <version> and
+# <link> for a given plugin <name> (if it exists).  If we find a match, we'll
+# write it to a temporary files and read it back afterwards.
+VERSION_FILE="/tmp/plugin-version-$(uuidgen)"
+LINK_FILE="/tmp/plugin-link-$(uuidgen)"
+(
+python3 - <<PYTHON_SCRIPT
+
+import bs4
+import requests
```

Comment:
This depends on the python packages [bs4](https://pypi.org/project/beautifulsoup4/) and [requests](https://pypi.org/project/requests) being installed on the user's system.  In general, these can be installed via
```bash
$ python3 -m pip install --user bs4 requests
```